src/components/CollectionDetails.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchArtworksByMaker } from "../services/apiService";
+
+function CollectionDetails() {
+  const { maker } = useParams();
+  const [artworks, setArtworks] = useState([]);
+
+  useEffect(() => {
+    const getArtworks = async () => {
+      const allArtworks = await fetchArtworksByMaker(decodeURIComponent(maker));
+      setArtworks(allArtworks);
+    };
+    getArtworks();
+  }, [maker]);
+
+  return (
+    <div>
+      <h1>Artworks by {decodeURIComponent(maker)}</h1>
+      {artworks.map((artwork) => (
+        <div key={artwork.id}>
+          <h2>{artwork.title}</h2>
```

Comment:
Here I am able to see the images but not the text that is describing about the images.

Reply:
Resolved in my latest commit, thanks!